src/pretix/settings.py

```diff
@@ -116,6 +116,29 @@
 
 db_options = {}
 
+postgresql_sslmode = config.get('database', 'sslmode', fallback='disable')
+USE_DATABASE_TLS = postgresql_sslmode != 'disable'
+USE_DATABASE_MTLS = config.has_option('database', 'sslcert')
+
+if USE_DATABASE_TLS or USE_DATABASE_MTLS:
+    tls_config = {}
+    if USE_DATABASE_TLS:
+        if 'postgresql' in db_backend:
+            tls_config = {
+                'sslmode': config.get('database', 'sslmode'),
+                'sslrootcert': config.get('database', 'sslrootcert'),
+            }
+    else:
+        if 'postgresql' in db_backend:
+            tls_config = {
+                'sslmode': config.get('database', 'sslmode'),
+                'sslrootcert': config.get('database', 'sslrootcert'),
+                'sslcert': config.get('database', 'sslcert'),
+                'sslkey': config.get('database', 'sslkey'),
+            }
+
+    db_options.update(tls_config)
```

Comment:
Isn't this the wrong way around?

```suggestion
USE_DATABASE_TLS = postgresql_sslmode != 'disable'
USE_DATABASE_MTLS = USE_DATABASE_TLS and config.has_option('database', 'sslcert')

if USE_DATABASE_TLS or USE_DATABASE_MTLS:
    tls_config = {}
    if not USE_DATABASE_MTLS:
        if 'postgresql' in db_backend:
            tls_config = {
                'sslmode': config.get('database', 'sslmode'),
                'sslrootcert': config.get('database', 'sslrootcert'),
            }
    else:
        if 'postgresql' in db_backend:
            tls_config = {
                'sslmode': config.get('database', 'sslmode'),
                'sslrootcert': config.get('database', 'sslrootcert'),
                'sslcert': config.get('database', 'sslcert'),
                'sslkey': config.get('database', 'sslkey'),
            }

    db_options.update(tls_config)
```

Reply:
Done